pkg/provider/resource_cluster.go

```diff
@@ -66,6 +66,8 @@ type ClusterResourceModel struct {
 	AllowedIpRanges            []AllowedIpRangesResourceModel     `tfsdk:"allowed_ip_ranges"`
 	CreatedAt                  types.String                       `tfsdk:"created_at"`
 	MaintenanceWindow          *commonTerraform.MaintenanceWindow `tfsdk:"maintenance_window"`
+	ServiceAccountIds          types.List                         `tfsdk:"service_account_ids"`
```

Comment:
Better to use types.Set. List have to persist order and the response API does not persist order which makes list difficult to maintain as you may have to sort it in many places

Reply:
done.